### DISCUSSION TITLE
bootcommerce out of date template file

### DISCUSSION THREAD

**Discussion by bamhost** [bootscore/bootscore#492] [Q&A]
This is regarding the BootCommerce template. For a few weeks I've been having the notice that the order-details.php is out of date. I've not received any notices from BootScore that a template update has been released. I checked the version of order-details.php in my installs of the BootScore Main and see the version of the order-details.php is 4.7.0. I've tried several times to go to bootscore.me to download the latest version in hopes that this would be updated in it, but the website never loads. Even now as I type this, over 5 minutes in, and the site is still trying to load. All that is on the screen is the spinning loader icon.

So this leads me to questions... do we need to be worried about future updates of the bootscore templates? Is there somewhere else that we should be going to download current versions of your templates?

Thank you for your help.

**Comment by justinkruit** [comment]
Hello,

Seems like the website is working fine from here. If you'd like to download the latest version, you can do so from our GitHub: https://github.com/bootscore/bootscore/releases

We also removed the order-details.php 3 months ago already from our theme as we don't use that template anymore for styling. This was part of #420 and is integrated in the [v5.2.3.3](https://github.com/bootscore/bootscore/releases/tag/v5.2.3.3) update.
